action.yml

```diff
@@ -20,6 +20,13 @@ inputs:
   generate_attestations:
     description: "Whether to generate artifact attestations for release binaries to establish build provenance, defaults to `false` if unspecified"
     default: false
+  release_android:
+    description: "Whether to release android binaries, defaults to `false` if unspecified"
+    default: false
+  android_ndk_home:
+    description: "Path to the Android NDK for android-amd64 and android-arm64 builds, required for Go 1.22 or newer, defaults to `ANDROID_NDK_HOME` environment variable if unspecified"
```

Comment:
This is kinda long for an input description 🤔

How much of this info should be here versus on the readme?  Also, should we mention that GitHub-managed runners have this installed by default?

If we do include info on the readme, perhaps we should link to the doc sections of https://github.com/actions/runner-images for users to look up what versions come installed.

Reply:
Agreed. I trimmed it down and opted to have more of this information in the readme. I added the link you suggested as well.